docs/cody/usage-and-pricing.mdx

```diff
@@ -47,6 +56,28 @@ To revert back to Cody Free from Pro:
 
 To upgrade from Cody Pro to Cody Enterprise, you should [Contact Sales](https://sourcegraph.com/contact/request-info) and connect with one of our account teams. They will help you set up your account and start with Cody Enterprise.
 
+## Billing FAQs for Cody Pro
+
+### What is Cody's pricing plan?
+
+You can get this info from our [pricing page](https://sourcegraph.com/pricing?product=cody).
+
+### When are payments taken?
+
+We charge payments at the beginning of each billing cycle, and they get automatically renewed once you've added your credit card details.
+
+### What is a billing cycle?
+
+The billing cycle refers to the start date and depends on your account creation date. You can view your current and previous billing cycles from the **My subscription** tab.
```

Comment:
The billing cycle doesn't depends on the account creation date. It depends only on when the user starts their Cody Pro plan.

Reply:
I think we should also reference something like "Your Cody Pro plan will automatically renew at each billing date."